integration-tests/src/test/java/io/quarkus/qe/RepositoryWorkflowIT.java

```diff
@@ -45,6 +53,10 @@ private void thenRepositoryShouldBeUpdated(String expectedRepoUrl, String expect
             assertEquals(expectedName, actual.getName());
             assertNotNull(actual.getExtensions());
             assertFalse(actual.getExtensions().isEmpty());
+            assertEquals(1, actual.getLabels().size());
+            assertEquals(LABEL, actual.getLabels().iterator().next());
+            assertNotNull(actual.getUpdatedTimestamp());
+            assertEquals("COMPLETED", actual.getStatus());
```

Comment:
If this literal match and refer to RepositoryStatus Completed Enum, could we use the Enum as a String?

Reply:
I didn't want to import the `data` module into the integration tests for this. But I will add some constants to avoid this.